themes/default/content/what-is/aws-s3-cp.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Run 'aws s3 cp' using Dynamic Credentials
+meta_desc: |
+     Learn more about leveraging dynamic credentials in Pulumi ESC to run commands like aws s3 cp.
+
+type: what-is
+page_title: Run 'aws s3 cp' using Dynamic Credentials
+---
+
+Amazon Web Services (AWS) is a leader in cloud computing, transforming the way organizations manage their digital infrastructure. An important part of the cloud computing landscape is the management of secrets and configurations. The effective management of cloud credentials can pose a significant challenge, especially when it comes to managing them at scale. [Pulumi ESC (Environments, Secrets, and Configurations)](/docs/pulumi-cloud/esc/) is a service that helps to alleviate the burden of managing cloud configuration and secrets by providing a centralized way to handle these critical aspects of cloud development. The `esc run` command of this service in particular helps to resolve concerns around how to:
```

Comment:
i think the definition of AWS is fine. We should also then define S3 with a sentence (take it from their website or wikipedia). we should say copying files to and from S3 buckets is very common operation performed on the CLI. However the getting the right credentials into the CLI can be manual if you use temp credentials or insecure if you are using long term creds. ESC is a service that manages all your secrets (have a longer more general description of it). Specifically it can manage dynamic credentials from AWS using OIDC. This makes CLI operations seamless so you never have to worry about invalid credentials or copy pasting them around

Reply:
what you have here is very generic to overall value of ESC which is great, but not specifically when it comes to calling these commands from the AWS CLI